docs/ca/dev/best-practices/jenkins-stability-checklist.md

```diff
@@ -7,46 +7,53 @@ redirect_from:
 last_updated: March 11, 2024
 ---
 
-This checklist will help you implement Spryker’s best practices, enhancing the stability and performance of the Jenkins component in your Spryker PaaS environment. Before raising issues about Jenkins performance and stability with Spryker, make sure you have fully completed the check list. If you have concerns or questions about iy, raise them with Spryker Support.
-
-
-- [ ] Configure a maximum of 2 executors.
-- [ ] Set your PHP memory_limit to be less than 2GB.
-- [ ] Implement batch processing (guidelines are here and here) in your importers and be aware of the maximum memory consumption.
-- [ ] Fine-tune the chunk size of the queues you work with.
-- [ ] Ensure that your theoretical maximum memory demand (for all planned parallel processes) remains below the memory allocation of your Jenkins instance.
-- [ ] Verify that every PHP job you run consumes less memory than your specified PHP memory limit. No “PHP Fatal error: Out of memory” errors should occur.
-- [ ] Ensure that no jobs are configured with a non-default memory limit or without any memory limit at all in jenkins.php (e.g., php -d memory_limit=-1 vendor/bin/console ...).
-- [ ] Avoid spawning an excessive number of workers (not more than 2 per queue).
-- [ ] Profile your jobs locally to understand their normal memory demand, especially when interacting with data.
-- [ ] In a standard-sized non-production environment, run lengthy imports and sync processes lasting more than 1-2 hours.
-- [ ] Refer to troubleshooting instructions for further assistance.
-- [ ] Be prepared to lose manually created jobs; ensure that all critical jobs are persisted in your project (jenkins.php).
+This checklist is designed to help you implement Spryker’s best practices, to enhance the stability and performance of the Jenkins component in your Spryker PaaS environment. Before raising issues about Jenkins performance and stability with Spryker, make sure you have fully completed the checklist. If you have concerns or questions about it, raise them with Spryker Support.
+
+
+- Configure a maximum of two executors.
+- Set your PHP `memory_limit` value to be less than 2 GB.
+- Implement batch processing (guidelines are here and here) in your importers and be mindful of the maximum memory consumption.
```

Comment:
@ThomasLehnerSpryker please add the links to guidelines

Reply:
https://docs.spryker.com/docs/dg/dev/data-import/202311.0/data-import-optimization-guidelines.html
https://docs.spryker.com/docs/dg/dev/integrate-and-configure/integrate-elastic-computing.html